src/cz/zipek/sqflint/linter/Options.java

```diff
@@ -96,6 +99,36 @@ public Options() throws IOException {
 		operators.put("||", operators.get("or"));
 	}
 	
+	/**
+	 * Creates a copy of the options
+	 */
+	public Options(Options old) {
+		this.stopOnError = old.stopOnError;
+		this.skipWarnings = old.skipWarnings;
+		this.jsonOutput = old.jsonOutput;
+		this.outputVariables = old.outputVariables;
+		this.exitCodeEnabled = old.exitCodeEnabled;
+		this.warningAsError = old.warningAsError;
+		this.checkPaths = old.checkPaths;
+		this.contextSeparationEnabled = old.contextSeparationEnabled;
+		this.benchLogs = old.benchLogs;
+		this.rootPath = old.rootPath;
+		
+		this.outputFormatter = new TextOutput();
```

Comment:
Shouldn't the output formatter also be copied? That's what I'd expect at least...

Reply:
Good catch. Gonna add it to the constructor, since there is no (good) way to copy the output formatters (cuz inheritance and interface stuff)

They need to be provided on a per file basis in case of the server output